website/content/en/docs/building-operators/ansible/tutorial.md

```diff
@@ -235,7 +237,6 @@ Ensure that the memcached operator creates the deployment for the sample CR with
 ```console
 $ kubectl get deployment
 NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
-memcached-operator-controller-manager   1/1     1            1           8m
 memcached-sample                        3/3     3            3           1m
```

Comment:
The deployment will show up, but the sample will not
```suggestion
memcached-operator-controller-manager   1/1     1            1           8m
```

Reply:
Is this supposed to show the sample that is deployed with its 3 instances and hence in the default namespace? Earlier in the doc you show the operator using the command: `kubectl get deployment -n memcached-operator-system`.